vim_binding.js

```diff
@@ -399,6 +402,7 @@ define([
       'esc': 'ipython.close-pager',
       'q': 'ipython.close-pager',
       'enter': 'ipython.enter-edit-mode',
+      'f': 'ipython.find-and-replace',
```

Comment:
[SHOULD]
It does not exist. Please create an alternative action for IPython 4.x